modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/ElixirClientCodegen.java

```diff
@@ -50,10 +50,10 @@
     // This is the name of elixir project name;
     protected static final String defaultPackageName = "openapi_client";
 
-    String supportedElixirVersion = "1.4";
+    String supportedElixirVersion = "1.8";
```

Comment:
tesla requires `1.4`, poison requires `1.6` - I would suggest we just stick to 1.6 instead of 1.8 to allow higher backwards compatibility since 1.8 isn't really a requirement